#ifndef OBSTACLE_DETECTION_DANGERZONESERIALIZERTEST_H
#define OBSTACLE_DETECTION_DANGERZONESERIALIZERTEST_H

#include <gtest/gtest.h>

class DangerZoneSerializerTest : public ::testing::Test {

protected:

    DangerZoneSerializerTest();


};


#endif //OBSTACLE_DETECTION_DANGERZONESERIALIZERTEST_H
